### ISSUE TITLE
Ability to Handle Multiple URLs in the Admin

### ISSUE THREAD

**Issue by shize2wagto** [shiwaforce/total-tutorial#8]
Dear team,

I just wanted to say that your product is great! However, I have a feature request to suggest. When I was working on the admin page, I noticed that I was only able to configure one page at a time, even though the tutorial suggests that it is possible to configure multiple pages. Unfortunately, I can only save another page in a new config.

I was wondering, would it be possible to add a feature that allows me to upload an existing configuration to the admin? This way, I would be able to edit the parts related to the current page without affecting the other parts. Of course, it would be even better if I could simply compile the tutorial for the second page and have the first page's data saved automatically.

In summary, it would be great if the admin could support handling multiple URLs. Thank you for considering my suggestion.

Best regards
